src/clj/rems/handler.clj

```diff
@@ -15,6 +15,7 @@
             [rems.css.styles :as styles]
             [rems.db.catalogue :as catalogue]
             [rems.email.core] ;; to enable email polling
+            [rems.application.cleaner] ;; to enable draft application clean-up job
```

Comment:
I wonder if it would make more sense to refer to these (states) in e.g. handler (state) (artificially) so that it's obvious that they are needed and this comment becomes unnecessary?

Reply:
i'm not 100% sure i understood 😄  this init was a little difficult to place as i thought at first that we would have some unified place where all scheduler jobs would be initialized. do you mean perhaps that there should be a function that is called, such as `init-scheduler`?

Reply:
Now we require the namespace to initialize the scheduler, when the handler namespace is required and mounted. That makes it mount too, automatically.

But this comment tells me it's a subtle interaction that could accidentally be removed. We don't need the scheduler var anywhere, it is sort of self-contained otherwise. We could artificially refer to it somewhere to get it started and that would be more explicit than requiring just the namespace with comment.

We can also manually `mount/start` the scheduled jobs, like in `standalone.clj`. That would also be explicit. I think it's perhaps not the correct responsibility of the route handler anyway, but rather the whole application. And it could be nice sometimes to not start them. I think we need to add the manual startup in a fixture for some tests but it would not be too much work.

This smells like another ticket to create.

Reply:
yeah i would think so too. it would make more sense to load up all scheduled jobs in the same place

Reply:
New ticket is here https://github.com/CSCfi/rems/issues/2730.